workerToInline.js

```diff
@@ -23,7 +23,7 @@ fs.readFile(source, 'UTF-8', function (err, data) {
     let sourceCode = jsStringEscape(data);
 
     let outCode = `
```

Comment:
I'm very concerned about using template strings and `let`/`const`, since not all bowsers that support web workers support these ES6 language features.